New version of click (8.x.x) may have been causing an issue with cog_translate

To re-create, run test: 
```
$ pytest -k test_cog_translate_valid
```

```
rio_cogeo/cogeo.py:289: in cog_translate
    with click.progressbar(wind, file=fout, show_percent=True) as windows:  # type: ignore
.direnv/python-3.7.3/lib/python3.7/site-packages/click/_termui_impl.py:98: in __enter__
    self.render_progress()
.direnv/python-3.7.3/lib/python3.7/site-packages/click/_termui_impl.py:220: in render_progress
    echo(self.label, file=self.file, color=self.color)
.
.
.
>       file.write(out)  # type: ignore
E       AttributeError: 'str' object has no attribute 'write'

.direnv/python-3.7.3/lib/python3.7/site-packages/click/utils.py:298: AttributeError
```

Locking down to `click==7.1.2` seems to work around the issue:

```
$ pip install click==7.1.2
$ pytest -k test_cog_translate_valid
```
```
=============================================================================== test session starts ===============================================================================
platform darwin -- Python 3.7.3, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
rootdir: /Users/gcihj/Projects/_tools/rio-cogeo
plugins: cov-2.12.1
collected 74 items / 67 deselected / 7 selected

tests/test_cogeo.py .......                                                                                                                                                 [100%]

================================================================================ warnings summary =================================================================================
../../../.pyenv/versions/3.7.3/lib/python3.7/importlib/_bootstrap.py:219
  /Users/gcihj/.pyenv/versions/3.7.3/lib/python3.7/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
    return f(*args, **kwds)

rio_cogeo/profiles.py:184
  /Users/gcihj/Projects/_tools/rio-cogeo/rio_cogeo/profiles.py:184: UserWarning: Non-standard compression schema: webp. The output COG might not be fully supported by software not build against latest libtiff.
    " supported by software not build against latest libtiff.".format(key)

-- Docs: https://docs.pytest.org/en/stable/warnings.html
================================================================== 7 passed, 67 deselected, 2 warnings in 1.66s =====
```

Maybe a simple fix for now is to modify `requirements.txt`:
https://github.com/cogeotiff/rio-cogeo/blob/b1c3692b5adaa456dbc565860e9fa26453c5ad10/requirements.txt#L1-L5

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # Release Notes
 
+# 2.2.3 (2021-06-18)
+
+* use opened file for click progressbar (https://github.com/cogeotiff/rio-cogeo/pull/204)
+
 # 2.2.2 (2021-06-01)
 
 * Add dictionary access to `Info` model (author @geospatial-jeff, https://github.com/cogeotiff/rio-cogeo/pull/201)

--- a/rio_cogeo/cogeo.py
+++ b/rio_cogeo/cogeo.py
@@ -285,7 +285,8 @@ def cog_translate(  # noqa: C901
 
                 if not quiet:
                     click.echo("Reading input: {}".format(source), err=True)
-                fout = os.devnull if quiet else sys.stderr
+
+                fout = ctx.enter_context(open(os.devnull, "w")) if quiet else sys.stderr
                 with click.progressbar(wind, file=fout, show_percent=True) as windows:  # type: ignore
                     for _, w in windows:
                         matrix = vrt_dst.read(window=w, indexes=indexes)

